web/src/app/services/ServiceRules/ServiceRuleCreateDialog.tsx

```diff
@@ -158,12 +157,11 @@ export default function ServiceRuleCreateDialog(props: {
   const [actionsError, setActionsError] = useState<boolean>(false)
   const [createRuleStatus, commit] = useMutation(mutation)
 
-  const [{ fetching, error, data }] = useQuery({
+  const [{ fetching, data }] = useQuery({
     query,
     variables: {},
   })
   if (fetching && !data) return <Spinner />
-  if (error) return <GenericError error={error.message} />
```

Comment:
We could perhaps pass this error to the FormDialog so this doesn't get missed in any edge cases:
`errors={nonFieldErrors(createRuleStatus.error || error)}`